metadata-ingestion/src/datahub/ingestion/source/usage/usage_common.py

```diff
@@ -112,6 +116,7 @@ def make_usage_workunit(
 
 class BaseUsageConfig(BaseTimeWindowConfig):
     top_n_queries: pydantic.PositiveInt = 10
+    ignore_user_emails_from_usage: Optional[List[str]]
```

Comment:
Can we make this an AllowDenyPattern type to be consistent with rest of the pattern based filters we have?

Reply:
You are right, I changed it.